worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -156,7 +218,7 @@ private[celeborn] class Worker(
         pushServerLimiter,
         conf.workerPushHeartbeatEnabled,
         workerSource)
-    transportContext.createServer(conf.workerPushPort)
+    transportContext.createServer(conf.workerPushPort, getServerBootstraps(transportConf))
```

Comment:
We will need separate securedPorts for push and fetch as well.

Reply:
I was thinking the same while reviewing ...